src/api/services/ListingItemActionService.ts

```diff
@@ -70,6 +72,67 @@ export class ListingItemActionService {
         const itemTemplateModel = await this.listingItemTemplateService.findOne(data.listingItemTemplateId, true);
         const itemTemplate = itemTemplateModel.toJSON();
 
+        const FRACTION_TO_COMPRESS_BY = 0.6;
```

Comment:
can we only compress images if it is necessary?

Reply:
We don't (unless I messed up), don't worry